pkg/ocm/ocm_client.go

```diff
@@ -0,0 +1,87 @@
+package ocm
+
+import (
+	"encoding/json"
+	"os"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	"github.com/openshift-online/ocm-sdk-go/logging"
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+	hivev1 "github.com/openshift/hive/apis/hive/v1"
+)
+
+type Client interface {
+	GetAWSAccountClaim(clusterID string) (*awsv1alpha1.AccountClaim, error)
+	GetClusterDeployment(clusterID string) (*hivev1.ClusterDeployment, error)
+}
+
+type client struct {
+	Conn   *sdk.Connection
+	logger *sdk.GlogLogger
+}
+
+func NewOcmClient(ocmConfigFile string) (Client, error) {
+	if ocmconfig := os.Getenv("OCM_CONFIG"); ocmconfig == "" {
```

Comment:
Didn't check, but this means that sending an empty string and not having an envvar forces the env to be empty?

Reply:
For the case OCM_CONFIG is not set, it will set the envVariable OCM_CONFIG with the configuration position that was specified in the function argument. And yes if you don't provide a configPath, and don't have the env set will leave everything empty, and the connection will fail

Reply:
Then should the error also catch that case?